chromium_src/components/policy/core/common/policy_types.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_POLICY_CORE_COMMON_POLICY_TYPES_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_POLICY_CORE_COMMON_POLICY_TYPES_H_
+
+// Inject a new POLICY_SOURCE_BRAVE_ORIGIN at the end so it doesn't affect other
+// integer values which could be stored in logs or sent via network
+// serialization
+#define POLICY_SOURCE_COUNT POLICY_SOURCE_BRAVE_ORIGIN, POLICY_SOURCE_COUNT
```

Comment:
couldn't this potentially overlap with some future policy that is added here? Is it persisted anywhere? Also any issue that there's no corresponding value in device_management_backend.proto? Maybe we should actually map it to 0 so it maps to `SOURCE_UNKNOWN` in the proto file and never overlaps with anything?